model/src/main/java/cz/muni/fi/pa165/team/match/TeamMatchFacade.java

```diff
@@ -195,4 +197,64 @@ public void deleteMatchGoal(UUID goalId)
         entityManager.flush();
     }
 
+    /**
+     * Retrieves all matches which have already been played.
+     *
+     * @return collection of all played matches
+     */
+    public Collection<TeamMatch> findAllPlayedMatches()
+    {
+        return teamMatchRepository.findAllPlayedMatches();
+    }
+
+    /**
+     * Retrieves all matches which are planned to be played, but have not
+     * already been played.
+     *
+     * @return collection of all planned but not played matches
+     */
+    public Collection<TeamMatch> findAllPlannedMatches()
+    {
+        return teamMatchRepository.findAllPlannedMatches();
+    }
+
+    /**
+     * Retrieves all scored goals by home team in the given match.
+     *
+     * @param matchId match which home team should score some goals a those should be retrieved
+     * @return all scored goals by home team in the required match
+     */
+    public Collection<TeamMatchGoal> findAllGoalByHomeTeamInMatch(UUID matchId)
```

Comment:
nestudoval jsem nijak dukladne co dělá, ale jestli se nepletu, tak volá dva dotazy... a já měl na mysli, že by to šlo napsat tak, že ve vysledku bude volat jen 1 dotaz